### PR TITLE
Hotfix to remove username prop

### PR DESCRIPTION
**Pull request by kurianbenoy-aot**
# Issue Tracking
Bug fix
JIRA: 

# Changes

- Remove userName prop to get the details from keycloak token details
- Update package-lock to fix security vulnerabilities

# How has the change been tested?

- Tested the change along with @abilpraju-aot  and ensured claim userName is working and localstorage details are being stored.

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: camunda-formio-tasklist-vue/src/components/mixins/TaskListMixin.vue [hunk 1]
```diff
@@ -8,5 +8,6 @@ import BaseMixin from './BaseMixin.vue';
 export default class TaskListMixin extends Mixins(BaseMixin) {
 
   public maximize = true;
+  public userName = '';
 }
```
Comment (ann-aot) [inline camunda-formio-tasklist-vue/src/components/mixins/TaskListMixin.vue:12]: try to maintain the variable format like "public/private userName: <type> = defaultvalue

**Review by ann-aot — APPROVED** [review]

File: camunda-formio-tasklist-vue/src/components/TaskList.vue [hunk 2]
```diff
@@ -195,7 +195,7 @@
                       <formio
                         :src="formioUrl"
                         :options="
-                          task.assignee === userName ? options : { readOnly: true }
+                          task.assignee !== userName ? { readOnly: true } : options
```
[the remark this answers is not in the document]
Comment (kurianbenoy) [reply camunda-formio-tasklist-vue/src/components/TaskList.vue:198]: Yes, options is defined as a variable in TaskList.vue component

[the remark this answers is not in the document]
Comment (krishnan-aot) [reply camunda-formio-tasklist-vue/src/components/TaskList.vue:198]: Is there a reason to flip the condition check here? The previous one does the same job.

File: camunda-formio-tasklist-vue/src/components/mixins/TaskListMixin.vue [hunk 3]
```diff
@@ -7,6 +7,7 @@ import BaseMixin from './BaseMixin.vue';
 })
 export default class TaskListMixin extends Mixins(BaseMixin) {
 
-  public maximize = true;
+  public maximize: boolean = true;
+  public userName: string = '';
```
Comment (krishnan-aot) [inline camunda-formio-tasklist-vue/src/components/mixins/TaskListMixin.vue:11]: Any reason why it is set up to empty string and not null?

Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/components/mixins/TaskListMixin.vue:11]: No particular reason, should we make it null @krishnan-aot ?

File: camunda-formio-tasklist-vue/src/components/TaskList.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/components/TaskList.vue:198]: Both ways, it worked. Just for consistency with overlay component above. I made this change

**Review by krishnan-aot — APPROVED** [review]

**Review by ann-aot — APPROVED** [review]

**Pull request merged**